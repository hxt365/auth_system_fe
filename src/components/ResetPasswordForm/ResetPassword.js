import React from 'react';

function ResetPasswordForm() {
  return <p>ResetPasswordForm</p>;
}

export default ResetPasswordForm;
