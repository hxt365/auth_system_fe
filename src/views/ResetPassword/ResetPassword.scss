.reset-password {
  text-align: center;
  padding-top: 5rem;

  &__link {
    display: block;
    margin-top: 1.5rem;
  }
}
