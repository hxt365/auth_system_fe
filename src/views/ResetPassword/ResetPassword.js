import React from 'react';
import './ResetPassword.scss';
import ResetPasswordForm from 'components/ResetPasswordForm';
import { Link } from 'react-router-dom';
import { LOGIN } from 'constants/index';

function ResetPassword() {
  return (
    <section className="reset-password">
      <h2>We will send you an email containing new password</h2>
      <ResetPasswordForm />
      <Link className="reset-password__link" to={LOGIN}>
        Remember out your password?
      </Link>
    </section>
  );
}

export default ResetPassword;
