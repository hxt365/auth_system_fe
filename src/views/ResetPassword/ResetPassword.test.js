import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import ResetPassword from './ResetPassword';

describe('test ResetPassword', () => {
  it('test snapshot', () => {
    const wrapper = shallow(<ResetPassword />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
