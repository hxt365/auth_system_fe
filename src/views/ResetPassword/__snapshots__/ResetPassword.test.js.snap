// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ResetPassword test snapshot 1`] = `
<section
  className="reset-password"
>
  <h2>
    We will send you an email containing new password
  </h2>
  <Form(ResetPasswordForm) />
  <Link
    className="reset-password__link"
    to="/login"
  >
    Remember your password?
  </Link>
</section>
`;
